docs/src/main/asciidoc/performance-measure.adoc

```diff
@@ -9,13 +9,34 @@ include::./attributes.adoc[]
 
 This guide covers:
 
+* Coordinated omission Problem in Tools
```

Comment:
I wouldn't have this as first bullet. In fact I don't think it belongs in this guide - no strong opinion about that, but at least let's not make this aspect too distracting from the primary content.